quick-start-with-tidb.md

```diff
@@ -57,10 +57,10 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
         {{< copyable "shell-regular" >}}
 
         ```shell
-        tiup playground v4.0.0 --db 2 --pd 3 --kv 3 --monitor
+        tiup playground v4.0.11 --db 2 --pd 3 --kv 3 --monitor
```

Comment:
@King-Dylan @morgo PTAL whether any version-specific change is needed here after 5.0 GA is released.